Source/Utils/RoomUtil.cs

```diff
@@ -94,6 +95,50 @@ private static object GetAula(HtmlNode? node)
         return new { name = nome, building = edificio, power = pwr, link = ROOM_INFO_URLS + info };
     }
 
+    public static async Task<object?> getRoomById(int id){
+        var url = ROOM_INFO_URLS + "Aula.do?" + 
+                  "idaula=" +  id;
+
+        var html = await HtmlUtil.DownloadHtmlAsync(url);
+        if (html.IsValid() == false) return null;
+        /*
+        example of property tag
+        <td colspan="1" rowspan="1" style="width: 33%" class="ElementInfoCard1 jaf-card-element">
+			<i>Codice vano</i>
+			<br>&nbsp;LCF040800S042
+		</td>
+        (parsing doesn't work very well, regex++)
+        */
+        string fetchedHtml = html.GetData() ?? "";
+        String[] fields = {"Sigla", "Capienza", "Edificio"};
+        //other fields include "Tipologia", "Indirizzo", "Dipartimento", "Codice vano", "Postazione per studenti disabili", ...
+        var properties = new Dictionary<String, String>();

```

Comment:
```suggestion
        var properties = new JObject();

```
Sarebbe meglio usare un `JObject`, così possiamo usare altri formati oltre a `<String>:<String>` nel JSON di risposta